spark8t/cli/service_account_registry.py

```diff
@@ -102,7 +107,13 @@ def create_service_account_registry_parser(parser: ArgumentParser):
 
     logging.basicConfig(format="%(message)s", level=args.log_level)
 
-    kube_interface = LightKube(args.kubeconfig or defaults.kube_config, defaults)
+    kube_interface = (
+        LightKube(args.kubeconfig or defaults.kube_config, defaults)
+        if args.backend == "lightkube"
+        else KubeInterface(
+            args.kubeconfig or defaults.kube_config, context_name=args.context
+        )
+    )
```

Comment:
Hmmm, instead of defining this 3 times in 3 modules, couldn't we add it to `Defaults`?